src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -165,157 +152,143 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
-                            "${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}", Const.ParamsNames.FEEDBACK_RESPONSE_TEXT,
-                            "${constSumOptionValue}", "");
+                            Slots.QUESTION_INDEX, Integer.toString(qnIdx),
+                            Slots.RESPONSE_INDEX, Integer.toString(responseIdx),
+                            Slots.OPTION_INDEX, "0",
+                            Slots.DISABLED, sessionIsOpen ? "" : "disabled",
+                            Slots.CONSTSUM_OPTION_VISIBILITY, "style=\"display:none\"",
+                            Slots.CONSTSUM_OPTION_POINT, existingConstSumResponse.getAnswerString(),
+                            Slots.FEEDBACK_RESPONSE_TEXT, Const.ParamsNames.FEEDBACK_RESPONSE_TEXT,
+                            Slots.CONSTSUM_OPTION_VALUE, "");
             optionListHtml.append(optionFragment).append(Const.EOL);
         } else {
             for (int i = 0; i < constSumOptions.size(); i++) {
                 String optionFragment =
                         Templates.populateTemplate(optionFragmentTemplate,
-                                "${qnIdx}", Integer.toString(qnIdx),
-                                "${responseIdx}", Integer.toString(responseIdx),
-                                "${optionIdx}", Integer.toString(i),
-                                "${disabled}", sessionIsOpen ? "" : "disabled",
-                                "${constSumOptionVisibility}", "",
-                                "${constSumOptionPoint}", Integer.toString(existingConstSumResponse.getAnswerList().get(i)),
-                                "${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}", Const.ParamsNames.FEEDBACK_RESPONSE_TEXT,
-                                "${constSumOptionValue}", Sanitizer.sanitizeForHtml(constSumOptions.get(i)));
+                                Slots.QUESTION_INDEX, Integer.toString(qnIdx),
+                                Slots.RESPONSE_INDEX, Integer.toString(responseIdx),
+                                Slots.OPTION_INDEX, Integer.toString(i),
+                                Slots.DISABLED, sessionIsOpen ? "" : "disabled",
+                                Slots.CONSTSUM_OPTION_VISIBILITY, "",
+                                Slots.CONSTSUM_OPTION_POINT, Integer.toString(existingConstSumResponse.getAnswerList().get(i)),
+                                Slots.FEEDBACK_RESPONSE_TEXT, Const.ParamsNames.FEEDBACK_RESPONSE_TEXT,
+                                Slots.CONSTSUM_OPTION_VALUE,  Sanitizer.sanitizeForHtml(constSumOptions.get(i)));
                 optionListHtml.append(optionFragment).append(Const.EOL);
             }
         }
 
-        String html = Templates.populateTemplate(
-                FeedbackQuestionFormTemplates.CONSTSUM_SUBMISSION_FORM,
+        return Templates.populateTemplate(
+                FormTemplates.CONSTSUM_SUBMISSION_FORM,
                 "${constSumSubmissionFormOptionFragments}", optionListHtml.toString(),
```

Comment:
I have just put those literals to Consts which we are using equal to more than 3 times. Should I change all literals to Consts ?